src/lib/permissions.js

```diff
@@ -14,7 +14,7 @@ const permit = async (userInfoURL, token, requestMethod, requestPath, log) => {
         agent: selfSignedAgent,
     };
     const userinfo = await fetch(userInfoURL, opts).then((res) => res.json());
-    log.info('Got userinfo:', userinfo);
+    log.info(`Got userinfo: ${userinfo}`);
```

Comment:
This will result in better structure in the logs (although it's less logger-agnostic).
```suggestion
    log.child({ userInfo }).info('Got userinfo');
```